bin/nefc

```diff
@@ -283,7 +321,7 @@ makeHeaders() {
     local playground=$2 # parameter `playgroundName`
     local output="nef/build/$playground.swift"
 
-    libs=(`xcrun swiftc -emit-imported-modules "$content" | sort -r`)
+    libs=(`xcrun swiftc -emit-imported-modules "$content" | sort -r | grep -vE PlaygroundSupport`)
```

Comment:
ignore `PlaygroundSupport` - we include a `NefPlaygroundSupport` to keep compatibility between `nef`- `osx` and `iOS`